portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro

```diff
@@ -771,6 +771,12 @@ definition {
 			locator1 = "Form#SIDEBAR_ADVANCED_PREDEFINED_VALUE");
 	}
 
+	macro setPlaceholder {
```

Comment:
@Diogomes, please check this [comment](https://github.com/liferay-forms/liferay-portal/pull/584#discussion_r652964541) and replicate it on this pull. :wink: 

Thanks.